Assets/Scripts/Level/DisableSpriteRender.cs

```diff
@@ -4,14 +4,14 @@
 
 public class DisableSpriteRender : MonoBehaviour
 {
-    public bool disableInEditor = false ; 
+    public bool disableInEditor = false;
+    public bool doNotDisableSpriteRenderer = false;
     void Start()
     {
-
+        if (doNotDisableSpriteRenderer) return;
```

Comment:
@MitchellCoon  for objects that have a Hitbox applied directly as a component instead of a child, we prob don't want to turn the sprite off in built versions of the game.

One example is the MissilePrefab. Hitbox is not a child of this object, but is directly applied to the prefab root. Currently, when the game is built, the sprite will be turned off for this object.

Reply:
To clarify, checking the `doNotDisableSpriteRenderer` will prevent this.